robotoff/app/api.py

```diff
@@ -1106,3 +1151,6 @@ def on_get(self, req: falcon.Request, resp: falcon.Response, username: str):
 api.add_route("/api/v1/health", HealthResource())
 api.add_route("/api/v1/dump", DumpResource())
 api.add_route("/api/v1/users/statistics/{username}", UserStatisticsResource())
+api.add_route(
+    "/api/v1/insights/predictions/display", DisplayInsightPredictionForProducts()
+)
```

Comment:
Why not just `"/api/v1/insights/predictions"` to be coherent with the url for insights ?